cfgov/cfgov/urls.py

```diff
@@ -412,7 +429,8 @@
         url(r'^(?i)obtener-respuestas/api/autocomplete-(?P<language>es)/$',
             ask_autocomplete, name='ask-autocomplete-es'),
     ]
-    urlpatterns += ask_patterns
+
+urlpatterns += ask_patterns
```

Comment:
@higs4281 currently none of these URLs overlap between the knowledgebase and ask_cfpb apps, so this code should work. However, if there are going to be URLs that overlap, we should talk about how to make that work.

In all my local testing, the flag works as expected, the knowledgebase URLs are available when `WAGTAIL_ASK_CFPB` is disabled, and the ask_cfpb URLs are available when `WAGTAIL_ASK_CFPB` is enabled.